src/server/src/services/github.js

```diff
@@ -31,27 +31,43 @@ const OctokitWithPluginsAndDefaults = Octokit.plugin(
     }
 })
 
-async function callGithub(octokit, obj, fun, arg, cacheKey, cacheTime) {
-    if (cacheKey && !config.server.nocache) {
-        const cachedRes = cache.get(cacheKey)
-        if (cachedRes) {
-            logger.info(`Result returned from cache for ${obj}.${fun}`)
-            return cachedRes
+async function callGithub(octokit, obj, fun, arg, cacheKey, isUseETag) {
+    let res
+
+    if (isUseETag && !config.server.nocache) {
+        const cachedData = cache.get(cacheKey)
```

Comment:
The cache currently uses an in-memory cache. 
cla-assistant.io currently can spawn for each process an own container with seperated memory. Therefore I would like the app to be as stateless as possible and use the mongodb as "cache".